src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -8,6 +8,15 @@ endif()
 
 set(TARGET_NAME "openvino_intel_cpu_plugin")
 
+#TODO: Patch KleidiAI CMakeLists to remove absolute include path
+execute_process(
```

Comment:
should it be done only if `ENABLE_KLEIDIAI_FOR_CPU` is enabled?

Reply:
it's going to be removed - I initiated kleidiai cmake duscussion with ACL team

Reply:
This workaround is removed since ACL team updates KleidiAI cmake